components/builder-api-client/src/lib.rs

```diff
@@ -218,41 +214,39 @@ pub struct OriginPrivateSigningKey {
 mod json {
     #[derive(Clone, Deserialize)]
     pub struct Package {
-        pub ident: PackageIdent,
+        pub ident:    PackageIdent,
         pub checksum: String,
         pub manifest: String,
-        pub deps: Vec<PackageIdent>,
-        pub tdeps: Vec<PackageIdent>,
-        pub exposes: Vec<u32>,
-        pub config: String,
+        pub deps:     Vec<PackageIdent>,
+        pub tdeps:    Vec<PackageIdent>,
+        pub exposes:  Vec<u32>,
+        pub config:   String,
     }
 
     #[derive(Clone, Deserialize)]
     pub struct PackageIdent {
-        pub origin: String,
-        pub name: String,
+        pub origin:  String,
+        pub name:    String,
         pub version: String,
         pub release: String,
     }
 
     impl From<PackageIdent> for super::PackageIdent {
         fn from(ident: PackageIdent) -> Self {
-            super::PackageIdent {
-                origin: ident.origin,
-                name: ident.name,
-                version: Some(ident.version),
-                release: Some(ident.release),
-            }
+            super::PackageIdent { origin:  ident.origin,
+                                  name:    ident.name,
+                                  version: Some(ident.version),
+                                  release: Some(ident.release), }
         }
```

Comment:
I think this is `indent_style = "Visual"` in action, and honestly, I could go either way on this.

Reply:
Yeah, both look fine to me